sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -0,0 +1,1188 @@
+  readonly name: string;
+  // (undocumented)
+  setAccessControl(
+    acl: PathAccessControlItem[],
+    options?: PathSetAccessControlOptions
+  ): Promise<PathSetAccessControlResponse>;
+  // (undocumented)
+  setHttpHeaders(
+    httpHeaders: PathHttpHeaders,
+    options?: PathSetHttpHeadersOptions
+  ): Promise<PathSetHttpHeadersResponse>;
+  // (undocumented)
+  setMetadata(
+    metadata: Metadata,
+    options?: PathSetMetadataOptions
+  ): Promise<PathSetMetadataResponse>;
+  // (undocumented)
+  setPermissions(
+    permissions: PathPermissions,
+    options?: PathSetPermissionsOptions
+  ): Promise<PathSetAccessControlResponse>;
+  // (undocumented)
+  toDirectoryClient(): DataLakeDirectoryClient;
+  // (undocumented)
+  toFileClient(): DataLakeFileClient;
+}
+
+// @public (undocumented)
+export interface DataLakeRequestConditions
+  extends ModifiedAccessConditions,
+    LeaseAccessConditions {}
+
+// @public (undocumented)
+export class DataLakeServiceClient extends StorageClient {
+  constructor(
+    url: string,
+    credential?: StorageSharedKeyCredential | AnonymousCredential | TokenCredential,
+    options?: StoragePipelineOptions
+  );
+  constructor(url: string, pipeline: Pipeline);
+  // (undocumented)
+  getFileSystemClient(fileSystemName: string): DataLakeFileSystemClient;
+  // (undocumented)
+  getUserDelegationKey(
+    startsOn: Date,
+    expiresOn: Date,
+    options?: ServiceGetUserDelegationKeyOptions
+  ): Promise<ServiceGetUserDelegationKeyResponse>;
+  // (undocumented)
+  listFileSystems(
+    options?: ServiceListFileSystemsOptions
+  ): PagedAsyncIterableIterator<FileSystemItem, ServiceListFileSystemsSegmentResponse>;
+}
+
+export { deserializationPolicy };
+
+// @public (undocumented)
+export interface FileAppendOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  conditions?: LeaseAccessConditions;
+  // (undocumented)
+  contentHash?: Uint8Array;
+}
+
+// @public (undocumented)
+export interface FileFlushOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  close?: boolean;
+  // (undocumented)
+  conditions?: DataLakeRequestConditions;
+  // (undocumented)
+  pathHttpHeaders?: PathHttpHeaders;
+  // (undocumented)
+  retainUncommittedData?: boolean;
+}
+
+// @public (undocumented)
+export interface FileReadHeaders {
+  // (undocumented)
+  acceptRanges?: string;
+  // (undocumented)
+  blobCommittedBlockCount?: number;
+  // (undocumented)
+  blobContentMD5?: Uint8Array;
+  // (undocumented)
+  blobSequenceNumber?: number;
+  // (undocumented)
+  cacheControl?: string;
+  // (undocumented)
+  clientRequestId?: string;
+  // (undocumented)
+  contentCrc64?: Uint8Array;
+  // (undocumented)
+  contentDisposition?: string;
+  // (undocumented)
+  contentEncoding?: string;
+  // (undocumented)
+  contentLanguage?: string;
+  // (undocumented)
+  contentLength?: number;
+  // (undocumented)
+  contentMD5?: Uint8Array;
```

Comment:
is `contentMD5` different from `blobContentMD5` above?

Reply:
Renamed `blobContentMD5` to `fileContentMD5`. They are not same. `ContentMD5` is md5 value for existing response body, while `fileContentMD5` is value for whole file.